x-pack/plugins/ml/public/datavisualizer/datavisualizer.html

```diff
@@ -29,63 +29,107 @@ <h1>{{indexPattern.title}}</h1>
               type="text"
               class="kuiLocalSearchInput"
               ng-model="searchQueryText"
-              placeholder="Search... (e.g. status:200 AND extension:PHP)"
-              aria-label="Search input"
+              placeholder="{{ ::'xpack.ml.datavisualizer.searchFieldPlaceholder' | i18n: {
+                defaultMessage: 'Search… (e.g. status:200 AND extension:PHP)'
+              } }}"
+              aria-label="{{ ::'xpack.ml.datavisualizer.searchFieldAriaLabel' | i18n: {
+                defaultMessage: 'Search input'
+              } }}"
               autocomplete="off"
             >
             <button
               class="kuiLocalSearchButton"
               type="submit"
-              aria-label="Search"
+              aria-label="{{ ::'xpack.ml.datavisualizer.searchButtonAriaLabel' | i18n: {
+                defaultMessage: 'Search'
+              } }}"
             >
               <span aria-hidden="true" class="kuiIcon fa-search"></span>
             </button>
           </div>
         </form>
 
         <div class="document-count-container">
-          <label for="sampleShardSize">Sample</label>
+          <label
+            for="sampleShardSize"
+            i18n-id="xpack.ml.datavisualizer.sampleLabel"
+            i18n-default-message="Sample"
+          ></label>
           <select id="sampleShardSize" class="kuiSelect type-input" ng-model="samplerShardSize" ng-change="samplerShardSizeChanged(samplerShardSize)">
             <option ng-value="1000">1000</option>
             <option ng-value="5000">5000</option>
             <option ng-value="10000">10000</option>
             <option ng-value="100000">100000</option>
-            <option ng-value="-1">all</option>
+            <option
+              ng-value="-1"
+              i18n-id="xpack.ml.datavisualizer.allOptionLabel"
+              i18n-default-message="all"
+            ></option>
           </select>
-          <span>documents per shard from a total of</span>
-          <span class="field-emphasis">{{ overallStats.totalCount }}</span>
-          <ng-pluralize count="overallStats.totalCount" when="{'1':'document', 'other':'documents'}">documents</ng-pluralize>
-          <i aria-hidden="true" class="fa fa-info-circle" tooltip-placement="right"
-            tooltip="Selecting a smaller sample size will reduce query run times and the load on the cluster."></i>
+          <span
+            i18n-id="xpack.ml.datavisualizer.documentsPerShardDescription"
+            i18n-default-message="documents per shard from a total of {wrappedTotalCount} {totalCount, plural, one {document} other {documents}}"
+            i18n-values="{
+              wrappedTotalCount: '<span class=\'field-emphasis\'>' + overallStats.totalCount + '</span>',
```

Comment:
Looks like the `<span>` tag shouldn't be used inside the `i18n-values` value:

![image](https://user-images.githubusercontent.com/7405507/49738718-f8d08600-fc87-11e8-8360-5935f0852c0d.png)

Reply:
With the last **master** <code>\<span\></code>  tag render correctly(using <code>overallStats.totalCount.value </code>)
![image](https://user-images.githubusercontent.com/25555662/50217780-c404b300-039a-11e9-97b2-fe5b18382103.png)